lisa/tools/modprobe.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from lisa.executable import Tool
+from lisa.util.process import ExecutableResult
+
+
+class Modprobe(Tool):
+    @property
+    def command(self) -> str:
+        return self._command
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._command = "modprobe"
+
+    def load_module(
```

Comment:
What's the method is used for? It does nothing almost. But if the return value can be parsed, it's good to have this method, and return structured result.

Reply:
`load_moule` runs modprobe. I have been using it for testing and figured that it should be left in the tool since it is the main functionality of modprobe. The command itself usually has no output but I thought maybe the user of the tool would want to check if the command had failed for any reason.

Reply:
Hmmm, I didn't see the reason to keep this method. The `is_module_loaded` is full functional. If there is any failure on modprobe itself, it should be detected by the is_module_loaded too. As a tool in LISA, the exists and installation is handled in the base classes. If there is different logic on checking existence or installation, the tool should overrides the default behavior.
In summary, it doesn't need this method to perform checks. It's the same as `run` of the tool.